db/migrations/00001_add_ff_task_metadata_table.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+-- Migration to create the ff_task_metadata table and its index.
+
+CREATE TABLE IF NOT EXISTS ff_task_metadata (
+    key                     VARCHAR(2048) NOT NULL PRIMARY KEY,
+    value                   TEXT,
+    marked_for_deletion_at  TIMESTAMP
+);
+
+CREATE INDEX IF NOT EXISTS ff_key_pattern
```

Comment:
You should probably remove these from metadata.go

Also, look for all "CREATE INDEX" "CREATE TABLE" references in the codebase, I think there's the ordered id generator and the lock tables as well right?